pkg/i2gw/providers/kong/consts.go

```diff
@@ -26,6 +31,23 @@ const (
 	pluginsKey = "plugins"
 )
 
+const (
+	v1beta1Version string = "v1beta1"
+
+	kongResourcesGroup gatewayv1beta1.Group = "configuration.konghq.com"
+
+	kongPluginKind gatewayv1beta1.Kind = "KongPlugin"
+	tcpIngressKind gatewayv1beta1.Kind = "TCPIngress"
```

Comment:
why gatewayv1beta1 and not just strings or kubernetes v1 kinds/groups?

Reply:
it was more convenient as we use these constants in gateway `ObjectRef`s. I agree that `string` fits better though. Changed.